pkg/volume/host_path/host_path.go

```diff
@@ -132,6 +121,9 @@ func (plugin *hostPathPlugin) NewDeleter(spec *volume.Spec) (volume.Deleter, err
 }
 
 func (plugin *hostPathPlugin) NewProvisioner(options volume.VolumeOptions) (volume.Provisioner, error) {
+	if !plugin.config.ProvisioningEnabled {
```

Comment:
It is highly useful for testing outside of a cloud environment.  That said, a mock plugin could be made to do the same thing. Swap out the plugins/provisioners in cmd based on the testing flag. 

This provisioner must be explicitly enabled for testing.